pkg/controller/installation/installer.go

```diff
@@ -54,7 +56,7 @@ func NewInstaller(chartFetchFunc shipperchart.FetchFunc,
 // cluster, or an error.
 func (i *Installer) renderManifests(_ *shipper.Cluster) ([]string, error) {
 	rel := i.Release
-	chart, err := i.fetchChart(rel.Spec.Environment.Chart)
+	chart, err := i.chartFetcher(&rel.Spec.Environment.Chart)
```

Comment:
We may or may not decide to do this in the scope of this PR, but it would be nice if the `chartFetcher` returned structured errors, so we could avoid wrapping everything with a `RenderManifestError`. I would expect some errors to be more retriable than others.